### PR TITLE
Add Quiz Category Page

### PR DESCRIPTION
**Pull request by varun13169**
image

### CODE REVIEW

File: src/page/quizCategoryPage/quiz-category-page.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    width: 50rem;
+    height: auto;
+    background-color: var(--quiz-app-quiz-select-card);
+    aspect-ratio: 1/1;
+}
+
+.quiz-card__img {
+    width: 100%;
+    height: 60%;
+    background-color: hotpink;
```
Comment (humzahasan) [inline src/page/quizCategoryPage/quiz-category-page.css:67]: Hope this is just for demo purpose

**Review by humzahasan — APPROVED** [review]
This PR have changes of last PR too! 

Code looks good to merge.

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                <div className="dui-nav-sch-act__search dui-inp-txt dui-inp-txt--search">
+                    <i className="dui-inp-txt__icon dui-inp-txt__icon--search">
+                        <svg xmlns="http://www.w3.org/2000/svg"
+                            className="dui-inp-txt__icon-content dui-inp-txt__icon-content--search icon icon-tabler icon-tabler-search"
+                            width="56" height="56" viewBox="0 0 24 24" stroke-width="1.5" stroke="#059669" fill="none"
+                            stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <circle cx="10" cy="10" r="7" />
+                            <line x1="21" y1="21" x2="15" y2="15" />
+                        </svg>
```
Comment (zoef-yt) [inline src/components/navbar/Navbar.jsx]: You can use [icones](https://icones.netlify.app/) to make lines of code less and increase reusability.

File: src/page/quizPage/QuizPage.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+                    <div class="quiz-card__status dui-util-spc-pad-s">
+                        <p class="quiz-card__ques-status dui-util-txt-align-left">Question: <span>1/5</span></p>
+                        <p class="quiz-card__score-status dui-util-txt-align-right">Score: <span>0</span></p>
+                    </div>
+
+                    <ul class="dui-ul quiz-card__options">
+                        <li class="quiz-card__option dui-util-bdr-radi-m dui-util-spc-pad-s">
+                            <p><span class="dui-util-fw-bld">1) </span>Option 1</p>
+                        </li>
```
Comment (zoef-yt) [inline src/page/quizPage/QuizPage.jsx]: Can be made into a component as you are repeating it.

**Review by zoef-yt — APPROVED** [review]
The code looks good. Do check out the suggestions I gave. Good work varun. Good to merge

File: src/components/navbar/Navbar.jsx (hunk 2 above)
Comment (varun13169) [reply src/components/navbar/Navbar.jsx]: Will make it part of refactoring.

File: src/page/quizCategoryPage/quiz-category-page.css (hunk 1 above)
Comment (varun13169) [reply src/page/quizCategoryPage/quiz-category-page.css:67]: Yup it'll be replaced by images

File: src/page/quizPage/QuizPage.jsx (hunk 3 above)
Comment (varun13169) [reply src/page/quizPage/QuizPage.jsx]: Yup, will make it part of refactoring.

**Pull request merged**